warehouse/internal/repo/staging_test.go

```diff
@@ -290,6 +290,21 @@ func TestStagingFileRepo_Many(t *testing.T) {
 			require.Nil(t, expectedSchemas)
 		})
 	})
+
+	t.Run("GetEventTimeRangesByUploadID", func(t *testing.T) {
```

Comment:
There is another scenario you're not testing. If not rows then it should return `nil, nil`.